### ISSUE TITLE
Invalid Column Name CreatedOn

### ISSUE THREAD

**Issue by FloteFuertes** [smartstoreag/SmartStoreNET#1259]
`protected override DbDataReader ExecuteDbDataReader(CommandBehavior behavior)
		{
			if (!IsCacheable)
			{
				**var result = _command.ExecuteReader(behavior);** --error here

				if (!_commandTreeFacts.IsQuery)
				{
					_cacheTransactionInterceptor.InvalidateSets(Transaction, _commandTreeFacts.AffectedEntitySets.Select(s => s.Name));
				}

				return result;
			}

			var key = CreateKey();

			object value;
			if (_cacheTransactionInterceptor.GetItem(Transaction, key, out value))
			{
				return new CachingDataReader((CachedRecords)value);
			}

			using (var reader = _command.ExecuteReader(behavior))
			{
				var queryResults = new List<object[]>();

				while (reader.Read())
				{
					var values = new object[reader.FieldCount];
					reader.GetValues(values);
					queryResults.Add(values);
				}

				return HandleCaching(reader, key, queryResults);
			}
		}`

**Comment by FloteFuertes** [comment]
CreatedOn keeps popping up is this an issue with EF?

**Comment by muratcakir** [comment]
Yes it's an EF issue in conjunction with MiniProfiler. And it's harmless. This exception is thrown by EF on purpose to perform some kind of fallback internally. Unfortunately MiniProfiler wraps some EF provider methods in such a way that the error pops up, but only during debug. There's nothing we or the MiniProfiler devs can do about it, other than getting rid of MiniProfiler of course, but we don't want that.

You can, however, suppress this error in Visual Studio: just deselect the exception type `System.Data.SqlClient.SqlException` in the _Exception settings_ window.

BTW: 
* This error only occurs when pending EF migrations are about to be executed
* It does not stop SmartStore from working properly. Just continue debugging and you're good.
